src/core/types.rs

```diff
@@ -105,7 +191,7 @@ pub struct Signature(pub Vec<u8>);
 pub type PublicKey = libp2p::identity::ed25519::PublicKey;
 pub type PrivateKey = libp2p::identity::ed25519::SecretKey;
 
-impl malachite_common::SigningScheme for Ed25519 {
+impl informalsystems_malachitebft_core_types::SigningScheme for Ed25519 {
```

Comment:
We will publish on crates.io (not yet done) the crate [informalsystems-malachitebft-signing-ed25519](https://github.com/informalsystems/malachite/blob/main/code/crates/signing-ed25519/Cargo.toml) that can be reused here potentially.